extensions/wikia/Email/Controller/FounderController.class.php

```diff
@@ -494,25 +497,25 @@ public function body() {
 	protected function getDetailsList() {
 		return [
 			[
-				"iconSrc" => Email\ImageHelper::getFileUrl( "Add_page.png" ),
+				"iconSrc" => Email\ImageHelper::getFileUrl( "AddPages.png" ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "iconSrc" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)